### ISSUE TITLE
Getting Started Page for Code Editor

### ISSUE THREAD

**Issue by 010101-sans** [face-hh/griddycode#76]
**Is your feature request related to a problem? Please describe.**
Yes, the problem is that new users of the code editor may find it challenging to understand and utilize its various features and functionalities. This can lead to a steep learning curve and potentially hinder the user experience.

**Describe the solution you'd like**
I propose the addition of a “Getting Started” page to the code editor. This page would serve as an introductory guide for new users, providing a list of various features and functionalities of the code editor. 

The “Getting Started” page could include the following sections:

1. **Introduction**: A brief overview of the code editor, its purpose, and its main features.

2. **Feature List**: A comprehensive list of the code editor’s features, with brief descriptions of each.

3. **Basic Operations**: A tutorial on how to perform basic operations in the code editor, such as creating a new file, saving a file, and editing a file.

4. **Advanced Features**: An overview of the code editor’s advanced features, such as key-bindings, code completion, syntax highlighting, and potentially version control integration.

5. **Troubleshooting**: A list of common issues that users might encounter and their solutions.

6. **Resources**: Links to additional resources, such as detailed documentation and Github repo link for new contributors.

This would make the code editor more user-friendly, particularly for new users, and enhance the overall user experience.

**Comment by QuinceTart10** [comment]
I assume you mean it kinda like the Visual Studio Code's welcome tutorial, right?

**Comment by 010101-sans** [comment]
Yes. But this code editor should retain it's original style and minimalism in getting started page too.
